src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -123,34 +123,36 @@ const CoverImageEditModal = ({
           Notification.Error({
             msg: "Something went wrong!",
           });
-          setIsUploading(false);
+          setIsProcessing(false);
         }
       },
       null,
       () => {
         Notification.Error({
           msg: "Network Failure. Please check your internet connectivity.",
         });
-        setIsUploading(false);
+        setIsProcessing(false);
       },
     );
 
     await sleep(1000);
```

Comment:
@AnkurPrabhu  can remove this and check if there is any sideeffects

Reply:
@khavinshankar can we connect on slack today i had some doubts.